src/DTO/Bandit/BanditEvaluationEvent.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Eppo\DTO\Bandit;
+
+use DateTime;
+use JsonSerializable;
+
+class BanditEvaluationEvent implements JsonSerializable
```

Comment:
many of the telemetry packages accept a dictionary/map-like structure for userdata.